README.md

```diff
@@ -2,4 +2,43 @@
 
 Cell Painting and other pipelines from the Imaging Platform.
 
-**Note**: The `without_batchfile` version of a pipeline is identical to the version without the suffix, except that the `CreateBatchFiles` module is disabled. The `without_batchfile` version exists because [Distributed-CellProfiler](https://github.com/CellProfiler/Distributed-CellProfiler) currently does not support the use of batch files. When updating the pipeline, ensure that these two versions are always in sync. 
+## Guidelines for creating pipelines 
+
+1. For each pipeline (e.g. `analysis.cpppipe`), always create a `without_batchfile` version of a pipeline (e.g. `analysis_without_batchfile.cpppipe`) and store in the same directory. The `without_batchfile` version is identical to the version without the suffix, except that the `CreateBatchFiles` module is disabled. The `without_batchfile` version exists because [Distributed-CellProfiler](https://github.com/CellProfiler/Distributed-CellProfiler) does not support the use of batch files. When updating the pipeline, ensure that these two versions are always in sync. 
+
+2. The `LoadData` module should always be configured with these values in order that the load data CSV file can be specified on command line
+```
+    Input data file location:Elsewhere...\x7C
+    Name of the file:
+```
+
+3. The `ExportToSpreadsheet` module should always be configured with these values in order that the files are saved in a standard format and location
+```
+    Select the column delimiter:Comma (",")
+    Add image metadata columns to your object data file?:No
+    Limit output to a size that is allowed in Excel?:No
+    Select the measurements to export:No
+    Output file location:Default Output Folder sub-folder\x7C
```

Comment:
Why would you want to export to a sub-folder but not then specify a sub folder? See SaveImages below for comparison.  All the cell painting pipelines i'm familiar with are set up like this- 
Output file location:Default Output Folder\x7C

Reply:
Makes sense - not sure why I was doing that